.github/workflows/update_deps.yml

```diff
@@ -0,0 +1,26 @@
+name: Monthly dependency updates
+on:
+  schedule:
+    # First of every month
+    - cron: 0 0 1 * *
+
+jobs:
+  create_issue:
+    name: Update dependencies
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Update dependencies
+        uses: imjohnbo/issue-bot@v3.3.4
```

Comment:
❗ GitHub had a list of things they recommend we do whenever using third party actions:

- Pin actions to a full length commit SHA

- Audit the source code of the action

- Pin actions to a tag only if you trust the creator

Ref: https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

Can you do the first two items?